pkg/settlement/swap/transaction/transaction.go

```diff
@@ -154,6 +154,13 @@ func (t *transactionService) WaitForReceipt(ctx context.Context, txHash common.H
 	}
 }
 
+func IncreaseGasPrice(gasPrice *big.Int) *big.Int {
```

Comment:
whatever changes you've made towards this file AND its test file don't seem highly necessary...